src/test_csv.py

```diff
@@ -0,0 +1,58 @@
+import dowel
```

Comment:
Can you update this test to use pytest instead, and put it in the appropriate place in `tests/`?

Reply:
No problem. Thank you for pointing out!